### PR TITLE
Review and edit Narwhal Docker README

### PR DESCRIPTION
**Pull request by Clay-Mysten**

### CODE REVIEW

File: Docker/README.md [hunk 1]
```diff
@@ -1,107 +1,111 @@
...
-authority is consisted of a `primary` node and one or more `worker` nodes. Since `Authority` is not a widely
-used term, we are referring instead to our `primary + worker` node pairs as `Validator`.
-_______
-
-The `logs` folder will be created once the node is bootstrapped via docker-compose. 
-For example, for the primary node of the validator-0, the logs will be found under
-the folder [logs](validators/validator-0/logs) and with the name `log-primary.txt`. To monitor the logging
-of a node in real time you can just do something like:
+
+> **Important:** You will find across the codebase the name *Authority* when we describe
```
Comment (huitseeker) [inline Docker/README.md:42]:
```suggestion
> **Note**: You will find across the codebase the name *Authority* when we describe
```

File: Docker/README.md [hunk 2]
```diff
@@ -1,107 +1,111 @@
...
 
-### Access worker node public gRPC endpoints
+## Access worker node public gRPC endpoints
 
-Similar to how someone can access the [public gRPC endpoints on a primary node](#access-primary-node-public-grpc-endpoints),
-**to feed transactions** to the Narwhal cluster via the `worker` nodes could be done via the gRPC server that is
-bootstrapped on the worker nodes bind to the local machine port. To send transactions the following local
+As you access the [public gRPC endpoints on a primary node](#access-primary-node-public-grpc-endpoints), you may
+similarly **feed transactions** to the Narwhal cluster via the `worker` nodes with the gRPC server
+bootstrapped on the worker nodes bond to the local machine port. To send transactions, the following local
```
Comment (huitseeker) [inline Docker/README.md:108]: Bind is used here as a technical term, not for its common English semantics.
```suggestion
bootstrapped on the worker nodes bind to the local machine port. To send transactions, the following local
```

**Review by huitseeker — COMMENTED** [review]
Thanks a bunch! 🙏

**Comment by Clay-Mysten** [comment]
> Thanks a bunch! 🙏

My pleasure! All done. :-)

**Review by akichidis — APPROVED** [review]
Awesome work @Clay-Mysten 🥇! Thanks so much!

**Pull request merged**